src/components/P5ChallengeTaskStepper/P5ChallengeTaskStepper.jsx

```diff
@@ -0,0 +1,146 @@
+          },
+        ],
+      },
+      {
+        stepTitle: "Finalize Component Together",
+        stepContent: [
+          {
+            sectionTitle: "Apply feedback",
+            description:
+              "• Collaborate with your partner to apply any feedback or improvements based on your reviews.",
+          },
+          {
+            sectionTitle: "Test the Final Version",
+            description:
+              "• Work together to test the final component and ensure everything functions correctly.",
+          },
+          {
+            sectionTitle: "Submit the Final Challenge",
+            description:
+              "• Once you are both satisfied with the final version of the component, submit the challenge by clicking Complete.",
+          },
+        ],
+      },
+    ],
+  };
+
+  const handleDashboard = () => {
+    navigate("/ChallengePage");
+  };
+  const [currentStep, setCurrentStep] = useState(0);
+
+  const handleNextStep = () => {
+    if (currentStep < 3) {
+      // Assuming you have 3 steps (you can adjust this value based on your step count)
+      setCurrentStep(currentStep + 1);
+    }
+  };
+  return (
+    // <div className="task-stepper-container">
+    //   {/* Connect with Partner Button */}
+    //   <div className="step-button">
+    //     <div className="button-text">Connect with Partner</div>
+    //   </div>
+
+    //   {/* Task Details */}
+    //   <div className="task-details">
+    //     {/* LinkedIn Section */}
+    //     <div className="task-section">
+    //       <div className="section-title">LinkedIn</div>
+    //       <div className="section-description">
+    //         Connect with your partner by using the LinkedIn button located in
+    //         the right tab next to “Connect.”
+    //         <br />
+    //         Alternatively, you can look them up by their username.
+    //       </div>
+    //     </div>
+
+    //     {/* Google Meet Section */}
+    //     <div className="task-section">
+    //       <div className="section-title">Google Meet</div>
+    //       <div className="section-description">
+    //         Use the Google Meet button in the right tab next to “Connect.” This
+    //         will direct you to Google Meet, where a meeting template will be
+    //         prefilled, making it easier to schedule a session with your partner.
+    //       </div>
+    //     </div>
+    //   </div>
+    // </div>
+    <div className="task-stepper-container">
+      {taskDetails.steps.map((step, index) => (
+        <>
+          <div key={index} className="step-section">
+            {/* Dynamic Button for each step */}
+            <div
+              className={`step-button ${currentStep == index ? 'completed' : ''}`}
```

Comment:
Currently Only the current step is green and the rest are gray. This could be improved with something like this: 

`className={`step-button ${currentStep > index ? 'completed' : currentStep === index ? 'active' : ''}`}`

the .active class would need to be added to the stylesheet with a blue background. This would make the current task blue, and the completed tasks green

Reply:
Fixed this